README.md

```diff
@@ -221,29 +227,31 @@ Las siguientes opciones de NixOS determinan qué certificados se aceptan o bloqu
 }
 ```
 
-Si `./mi-certificado.crt` está en la lista oficial, se incluirá en autofirma-nix.
-En cambio, dado que `Izenpe.com` se bloquea, se ignorará aunque aparezca en la lista oficial.
+If `./mi-certificado.crt` is on the official list, it will be included in autofirma-nix.
```

Comment:
`./my-certificado.crt` -> my-certificate.crt` for consistency.

Co-authored-by: @CesarGallego